TM1py/Services/RestService.py

```diff
@@ -788,6 +800,33 @@ def is_admin(self) -> bool:
 
         return self._is_admin
 
+    @property
+    def is_data_admin(self) -> bool:
+        if self._is_data_admin is None:
+            response = self.GET("/api/v1/ActiveUser/Groups")
```

Comment:
```suggestion
            response = self.GET("ActiveUser/Groups")
```

for compliance with v12 changes